### ISSUE TITLE
[nvmf/ns_hotplug_stress] nvmf_subsystem_remove_ns: subsystem busy, retry later

### ISSUE THREAD

**Issue by Jaylyn-Ren** [spdk/spdk#3079]
# CI Intermittent Failure
```
00:10:52.158  [2023-07-18 11:08:29.808930] ctrlr_bdev.c: 290:nvmf_bdev_ctrlr_read_cmd: *ERROR*: Read NLB 1 * block size 512 > SGL length 1
00:10:52.158  [2023-07-18 11:08:29.808978] ctrlr_bdev.c: 290:nvmf_bdev_ctrlr_read_cmd: *ERROR*: Read NLB 1 * block size 512 > SGL length 1
00:10:52.158  [2023-07-18 11:08:29.809019] ctrlr_bdev.c: 290:nvmf_bdev_ctrlr_read_cmd: *ERROR*: Read NLB 1 * block size 512 > SGL length 1
00:10:52.158  [2023-07-18 11:08:29.809076] ctrlr_bdev.c: 290:nvmf_bdev_ctrlr_read_cmd: *ERROR*: Read NLB 1 * block size 512 > SGL length 1
00:10:52.421  true
00:10:52.421   11:08:29	-- target/ns_hotplug_stress.sh@35 -- # kill -0 3593457
00:10:52.421   11:08:29	-- target/ns_hotplug_stress.sh@36 -- # /var/jenkins/workspace/nvmf-phy-autotest/spdk/scripts/rpc.py nvmf_subsystem_remove_ns nqn.2016-06.io.spdk:cnode1 1
00:10:52.421  request:
00:10:52.421  {
00:10:52.421    "nqn": "nqn.2016-06.io.spdk:cnode1",
00:10:52.421    "nsid": 1,
00:10:52.421    "method": "nvmf_subsystem_remove_ns",
00:10:52.421    "req_id": 1
00:10:52.421  }
00:10:52.421  Got JSON-RPC error response
00:10:52.421  response:
00:10:52.421  {
00:10:52.421    "code": -32603,
00:10:52.421    "message": "subsystem busy, retry later.\n"
00:10:52.421  }
00:10:52.421  [2023-07-18 11:08:30.148353] ctrlr_bdev.c: 290:nvmf_bdev_ctrlr_read_cmd: *ERROR*: Read NLB 1 * block size 512 > SGL length 1
00:10:52.421  [2023-07-18 11:08:30.148412] ctrlr_bdev.c: 290:nvmf_bdev_ctrlr_read_cmd: *ERROR*: Read NLB 1 * block size 512 > SGL length 1
00:10:52.421  [2023-07-18 11:08:30.148456] ctrlr_bdev.c: 290:nvmf_bdev_ctrlr_read_cmd: *ERROR*: Read NLB 1 * block size 512 > SGL length 1
00:10:52.421  [2023-07-18 11:08:30.148498] ctrlr_bdev.c: 290:nvmf_bdev_ctrlr_read_cmd: *ERROR*: Read NLB 1 * block size 512 > SGL length 1
[...]
00:10:52.427  in /var/jenkins/workspace/nvmf-phy-autotest/spdk/test/nvmf/target/ns_hotplug_stress.sh:36 -> main(["--transport=rdma"])
00:10:52.427  [2023-07-18 11:08:30.165935] ctrlr_bdev.c: 290:nvmf_bdev_ctrlr_read_cmd: *ERROR*: Read NLB 1 * block size 512 > SGL length 1
00:10:52.427       ...
00:10:52.427  [2023-07-18 11:08:30.165982] ctrlr_bdev.c: 290:nvmf_bdev_ctrlr_read_cmd: *ERROR*: Read NLB 1 * block size 512 > SGL length 1
00:10:52.427     31  	$SPDK_EXAMPLE_DIR/perf -c 0x1 -r "trtype:$TEST_TRANSPORT adrfam:IPv4 traddr:$NVMF_FIRST_TARGET_IP trsvcid:$NVMF_PORT" \
00:10:52.427     32  		-t 30 -q 128 -w randread -o 512 -Q 1000 &
00:10:52.427     33  	PERF_PID=$!
00:10:52.427     34  	
00:10:52.427     35  	while kill -0 $PERF_PID; do
00:10:52.427  => 36  		$rpc_py nvmf_subsystem_remove_ns nqn.2016-06.io.spdk:cnode1 1
00:10:52.427     37  		$rpc_py nvmf_subsystem_add_ns nqn.2016-06.io.spdk:cnode1 Delay0
00:10:52.427     38  		# Also test bdev/namespace resizing here, since it has a similar
00:10:52.427     39  		# effect on subsystem/namespace handling in the nvmf target
00:10:52.427     40  		null_size=$((null_size + 1))
00:10:52.427     41  		$rpc_py bdev_null_resize NULL1 $null_size
00:10:52.427       ...
00:10:52.427  
00:10:52.427  [2023-07-18 11:08:30.166027] ctrlr_bdev.c: 290:nvmf_bdev_ctrlr_read_cmd: *ERROR*: Read NLB 1 * block size 512 > SGL length 1
```
https://ci.spdk.io/results/autotest-per-patch/builds/107072/archive/nvmf-phy-autotest/build.log

**Comment by mikeBashStuff** [comment]
This affects nightly variant as well.

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @DamiaSan. log: https://ci.spdk.io/public_build/autotest-per-patch_107362.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @mikeBashStuff. log: https://ci.spdk.io/public_build/autotest-per-patch_107472.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @ksztyber. log: https://ci.spdk.io/public_build/autotest-per-patch_107487.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @apaszkie. log: https://ci.spdk.io/public_build/autotest-per-patch_107515.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @richael02. log: https://ci.spdk.io/public_build/autotest-per-patch_107739.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @mikeBashStuff. log: https://ci.spdk.io/public_build/autotest-per-patch_107853.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @mikeBashStuff. log: https://ci.spdk.io/public_build/autotest-per-patch_107852.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @tomzawadzki. log: https://ci.spdk.io/public_build/autotest-per-patch_107894.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @sberbz. log: https://ci.spdk.io/public_build/autotest-per-patch_107983.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @ksztyber. log: https://ci.spdk.io/public_build/autotest-per-patch_108898.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @ksztyber. log: https://ci.spdk.io/public_build/autotest-per-patch_109139.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @KKaras169169. log: Not Found

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @ksztyber. log: https://ci.spdk.io/public_build/autotest-per-patch_109273.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @sberbz. log: https://ci.spdk.io/public_build/autotest-per-patch_109800.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @mmkayPL. log: https://ci.spdk.io/public_build/autotest-per-patch_110006.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @tomzawadzki. log: https://ci.spdk.io/public_build/autotest-per-patch_110301.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @mmkayPL. log: https://ci.spdk.io/public_build/autotest-per-patch_110793.html

**Comment by tomzawadzki** [comment]
[Bug scrub] The `nvmf_subsystem_remove_ns` should be analyzed for valid reasons to return busy, and possibly the test should accommodate such condition.

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @karlatec. log: https://ci.spdk.io/public_build/autotest-per-patch_112280.html

**Comment by karlatec** [comment]
^different test case, but fail looks pretty similar to this issue

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @karlatec. log: https://ci.spdk.io/public_build/autotest-per-patch_112314.html

**Comment by tomzawadzki** [comment]
Unsure if related to this issue or new one, while working on enabling ASAN testing in multiple jobs I've got very similar error as in this issue.

https://ci.spdk.io/results/autotest-per-patch/builds/113351/archive/nvmf-phy-autotest_57801/build.log

Yet the new unique parts is that might be helpful while narrowing down this issue:
```
00:15:11.601  [2023-12-06 13:37:46.960206] subsystem.c: 437:spdk_nvmf_subsystem_destroy: *ERROR*: Subsystem can only be destroyed in inactive state, nqn.2016-06.io.spdk:cnode1 state 2
00:15:11.601  [2023-12-06 13:37:46.960275] nvmf.c: 469:nvmf_tgt_destroy_cb: *ERROR*: Failed to destroy subsystem nqn.2016-06.io.spdk:cnode1, rc -11
00:15:11.601  [2023-12-06 13:37:47.046832] bdev.c:2350:bdev_finish_unregister_bdevs_iter: *WARNING*: Unregistering claimed bdev 'NULL1'!
00:15:11.601  [2023-12-06 13:37:47.046970] thread.c:2585:spdk_for_each_channel: *ERROR*: could not find io_device 0x614000004040
00:15:11.601  nvmf_tgt: thread.c:2586: spdk_for_each_channel: Assertion `false' failed.
00:15:12.536  [2023-12-06 13:37:47.902481] nvme_qpair.c: 579:nvme_qpair_abort_queued_reqs: *ERROR*: aborting queued i/o
...
00:15:12.539  perf: perf.c:1479: submit_single_io: Assertion `!ns_ctx->is_draining' failed.
```

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @tomzawadzki. log: https://ci.spdk.io/public_build/autotest-per-patch_113928.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @mikeBashStuff. log: https://ci.spdk.io/public_build/autotest-per-patch_114504.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @ksprzacz. log: https://ci.spdk.io/public_build/autotest-per-patch_114713.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @apaszkie. log: https://ci.spdk.io/public_build/autotest-per-patch_114932.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @tomzawadzki. log: https://ci.spdk.io/public_build/autotest-per-patch_115026.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @mikeBashStuff. log: https://ci.spdk.io/public_build/autotest-per-patch_115384.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly_3236.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly_3233.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-main_3020.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-v21.11_3103.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-main_3015.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-v21.11_3099.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly_3224.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-v23.09-vs-dpdk-v22.11_385.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-v21.11_3112.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly_3241.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly_3242.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly_3243.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly_3244.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @mmkayPL. log: https://ci.spdk.io/public_build/autotest-per-patch_116138.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-v21.11_3119.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-v23.09-vs-dpdk-v22.11_395.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly_3253.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-main_3037.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly_3256.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @masuski. log: https://ci.spdk.io/public_build/autotest-per-patch_116243.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly_3258.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly_3261.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @mikeBashStuff. log: https://ci.spdk.io/public_build/autotest-per-patch_116410.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly_3270.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-main_3056.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @amichon-kalray. log: https://ci.spdk.io/public_build/autotest-per-patch_116477.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-main_3065.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-v24.01-LTS-vs-dpdk-v23.11_17.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly-lts_1421.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly_3287.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-v22.11_1423.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-v24.01-LTS-vs-dpdk-v22.11_28.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-v23.11_31.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-v24.01-LTS-vs-dpdk-v22.11_31.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly-lts_1451.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @ksprzacz. log: https://ci.spdk.io/public_build/autotest-per-patch_117192.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly-lts_1464.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly_3337.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly-lts_1476.html

**Comment by jimharris** [comment]
We need to fix `nvmf_subsystem_state_change()` to queue state change requests that come in while a state change is already in progress.

Please let's not "fix" the test to retry RPCs if one fails because of -EBUSY.

I think this should be fairly easy to reproduce if you start a target, and have 2 (or more) separate scripts running in a loop, just adding and removing a namespace from the same subsystem.

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly-lts_1482.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly-lts_1490.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly-lts_1498.html

**Comment by ksztyber** [comment]
> We need to fix nvmf_subsystem_state_change() to queue state change requests that come in while a state change is already in progress.
> I think this should be fairly easy to reproduce if you start a target, and have 2 (or more) separate scripts running in a loop, just adding and removing a namespace from the same subsystem.

You're right, I've just run two instances of add_ns/remove_ns in a loop and this issue reproduces after a few iterations (with a target running on 8 cores). I'll work on queuing subsystem state changes.

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly_3379.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly_3383.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly-lts_1514.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-v24.01-LTS-vs-dpdk-v22.11_112.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly-lts_1517.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-main_3166.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly_3401.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @jimharris. log: https://ci.spdk.io/public_build/autotest-per-patch_118143.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @mikeBashStuff. log: https://ci.spdk.io/public_build/autotest-per-patch_118165.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-main_3195.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly_3429.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly-lts_1555.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @apaszkie. log: https://ci.spdk.io/public_build/autotest-per-patch_118520.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly-lts_1560.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly_3440.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly-lts_1563.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly_3444.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-v23.11_176.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly_3455.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly-lts_1578.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly-lts_1583.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly_3474.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-v22.11_1594.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly-lts_1605.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-main_3270.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-v23.11_220.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly-lts_1621.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly-lts_1625.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-v24.01-LTS-vs-dpdk-v22.11_227.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly-lts_1633.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly-lts_1640.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-v24.01-LTS-vs-dpdk-v23.11_240.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-v24.01-LTS-vs-dpdk-v23.11_242.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-v24.01-LTS-vs-dpdk-v23.11_247.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly-lts_1654.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly-lts_1655.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly-lts_1657.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly-lts_1669.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-v23.11_270.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-v24.01-LTS-vs-dpdk-v23.11_269.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly-lts_1677.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @ksztyber. log: https://ci.spdk.io/public_build/autotest-per-patch_119809.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @shuhei-matsumoto. log: https://ci.spdk.io/public_build/autotest-per-patch_119974.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-v23.11_292.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly-lts_1701.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-v24.01-LTS-vs-dpdk-v22.11_299.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-v22.11_1699.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-v24.01-LTS-vs-dpdk-v23.11_303.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @KKaras169169. log: https://ci.spdk.io/public_build/autotest-per-patch_120329.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @ksztyber. log: https://ci.spdk.io/public_build/autotest-per-patch_120628.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @mikeBashStuff. log: https://ci.spdk.io/public_build/autotest-per-patch_120777.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly-lts_1736.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-main_3390.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-main_3394.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly_3627.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @ksztyber. log: https://ci.spdk.io/public_build/autotest-per-patch_120975.html

**Comment by ksztyber** [comment]
I've just pushed a patch that implements @jimharris's suggestion: https://review.spdk.io/gerrit/c/spdk/spdk/+/22942

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-main_3400.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-v24.01-LTS-vs-dpdk-v23.11_346.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-v24.01-LTS-vs-dpdk-v23.11_351.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-v22.11_1751.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-v23.11_356.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-v24.01-LTS-vs-dpdk-v22.11_355.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-v23.11_360.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly_3642.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-v22.11_1761.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-v24.01-LTS-vs-dpdk-v22.11_367.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-v24.01-LTS-vs-dpdk-v23.11_368.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly-lts_1782.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-v24.01-LTS-vs-dpdk-v22.11_379.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-v24.01-LTS-vs-dpdk-v22.11_382.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly-lts_1788.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-v24.01-LTS-vs-dpdk-v23.11_397.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly-lts_1811.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly-lts_1815.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly-lts_1831.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-v24.01-LTS-vs-dpdk-v22.11_427.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly-lts_1832.html

**Comment by mchomnic** [comment]
The fail still occurs, reopening

**Comment by ksztyber** [comment]
All of these failures occurred on the v24.01 job, which doesn't have 2881f7f69c3b14d43ed4411e677403595ee94cef. We could try to port it, although it requires porting several extra prep patches.

**Comment by ksztyber** [comment]
Patches have been merged onto the v24.01.x branch, closing.
